docs/diagrams/GoToUiClassDiagram.puml

```diff
@@ -0,0 +1,20 @@
+@startuml
+!include style.puml
+skinparam arrowThickness 1.1
+skinparam arrowColor UI_COLOR_T4
+skinparam classBackgroundColor UI_COLOR
+
+
+Class MainWindow
+
+MainWindow *-down-> TravelPlanObjectListPanel
+MainWindow *-down-> TravelPlanPanel
+MainWindow *-down-> TravelPlannerPanel
+
+TravelPlanPanel -down-> NameCard
```

Comment:
NameCard should be under TravelPlannerPanel

Reply:
I thought name card is the name of the travelplan, which is the travelplan panel? the one that shows the name and date

Reply:
No, its not. You go check the implementations in Ui component

Reply:
ok changed